src/main/kotlin/xyz/crossward/controller/GroupController.kt

```diff
@@ -27,28 +27,37 @@ class GroupController(
         return service.createWebsiteGroup(group)
     }
 
+    @PostMapping("/add_user")
```

Comment:
this should just be `/users` so the whole endpoint ends up being POST `/api/groups/users` You want to avoid verbs in the endpoints. The POST is the verb.